### PR TITLE
Added manual install instructions and fixed sub project

### PR DESCRIPTION
**Pull request by nrbrook**

### CODE REVIEW

**Comment by bpoplauschi** [comment]
Thanks @nrbrook

File: Documentation/GettingStarted.md [hunk 1]
```diff
@@ -8,16 +8,31 @@ There are 3 steps to getting started with the logging framework:
...
 
-The main files you need to add are:
+#### CocoaPods
 
--   DDLog (Basis of entire framework)
--   DDASLLogger (sends log statements to Apple System Logger, so they show up on Console.app)
--   DDTTYLogger (sends log statements to Xcode console - if available)
--   DDFileLogger (sends log statements to a file)
+```ruby
+platform :ios, '6.1'
```
Comment (rivera-ernesto) [inline Documentation/GettingStarted.md]: Shouldn't it be `'5.0'`?

Comment (nrbrook) [inline Documentation/GettingStarted.md]: This was copied from README.md. If it should be 5.0 (I'm not sure) it needs changing in both.

**Comment by rivera-ernesto** [comment]
I think it would be better to change both. Otherwise it gives the impression that iOS 5 is not supported.

**Pull request merged**